### PR TITLE
Rooms index page

### PR DESCRIPTION
**Pull request by wrpaape**
pls work

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -28,10 +28,20 @@ var ready = function() {
 
   var times = $('.timeDisplay');
   times.each(function () {
-    var time_text = $(this).html();
-    var formatted_time = moment(time_text).format('MMMM/DD/YYYY hh:mm a');
+    var time_text = $(this).html().format('MMMM/DD/YYYY hh:mm a');
```
Comment (jah2488) [inline app/assets/javascripts/application.js:31]: Did you mean to change these two lines?

**Comment by wrpaape** [comment]
:flames:

**Comment by BenHand** [comment]
:+1:

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (wrpaape) [inline app/assets/javascripts/application.js:31]: turns out yes, we're good now

**Comment by mlgiardina** [comment]
:neckbeard: :neckbeard: :neckbeard:

**Pull request merged**